live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/request/ATSTransformRequester.java

```diff
@@ -76,14 +81,23 @@ public void requestTransformRetry(TransformRequest transformRequest, int attempt
                 .sendBody(LOCAL_ENDPOINT, atsTransformRequest);
     }
 
-    private ATSTransformRequest toTransformRequest(TransformRequest transformRequest, int attempt)
+    protected ATSTransformRequest toTransformRequest(TransformRequest transformRequest, int attempt)
```

Comment:
I assume you switched to `protected` because you wanted to test this method?

Reply:
Yep